fuzz/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+
+[package]
+name = "dtb-fuzz"
+version = "0.0.1"
+authors = ["Automatically generated"]
+publish = false
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies.dtb]
+path = ".."
+[dependencies.libfuzzer-sys]
+git = "https://github.com/rust-fuzz/libfuzzer-sys.git"
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[[bin]]
+name = "reader"
+path = "fuzz_targets/reader.rs"
```

Comment:
Can't we redirect `corpus` to `test_dtb` not to duplicate the files?

Reply:
Maybe, and I looked it up for a while but didnt't a direct way to persistently configure `libfuzzer` so it didn't seem worth the benefit while maintaining the simple command line. The files are small by convention and not actually duplicated in git so it did not seem worth the extra hours.